### ISSUE TITLE
No Arm support?

### ISSUE THREAD

**Issue by Merith-TK** [Wonderfall/docker-mastodon#7]
![image](https://user-images.githubusercontent.com/10422110/202779655-f9b4128b-eaf5-43d8-9fcc-964a4dfea158.png)

**Comment by Wonderfall** [comment]
No arm64 support at the moment. Last time I tried adding support for it, I ran into some weird issues with Github Action runners, but that's a thing I'd like to solve eventually.

Until I find a solution, perhaps you could build the image yourself?

**Comment by Merith-TK** [comment]
I ended up just renting an amd64 server from [enviromc.host](https://client.enviromc.host/?affid=78) and got it setup, tried to rebuild with UID 1000 but that just... took forever.

now im running into the issue of there being no admin user

**Comment by Wonderfall** [comment]
> I ended up just renting an amd64 server from [enviromc.host](https://client.enviromc.host/?affid=78) and got it setup, tried to rebuild with UID 1000 but that just... took forever.

Yes, the image can take a significant amount of time to build. Lots of dependencies have to be either downloaded or built. It takes between 4 and 6 minutes in general for GitHub Actions to build the image. So it may take significantly longer on slower hardware.

> now im running into the issue of there being no admin user

Follow the official instructions: https://docs.joinmastodon.org/admin/setup/

**Comment by Merith-TK** [comment]
The Official Steps automatically create an admin user during setup process, but since there is no setup process, no admin user is created

EDIT: I actually clicked the link...

**Comment by Merith-TK** [comment]
okay how am I supposed to setup an admin user if I cannot access the data for the system?

**Comment by Wonderfall** [comment]
Not sure what you mean. You should be able to create an account from your browser (then you can make it admin) or from the command line (it should be like `docker exec -ti mastodon tootctl ...`). Your Mastodon container should have persistent volumes and access to a working database.
